velox/docs/develop/aggregate-functions.rst

```diff
@@ -248,6 +271,9 @@ For aggregaiton functions of default-null behavior, the author defines an
     // Author defines data members
     ...
 
+    // Define a pointer to the UDAF class.
```

Comment:
nit: Add in the comment that this is only needed if the aggregation function uses function-level states.